src/databricks/labs/ucx/migration/azure_sp.py

```diff
@@ -0,0 +1,162 @@
+import csv
+import io
```

Comment:
Rename file to azure.py or azure_credentials.py

Reply:
done